src/components/ui/sign-up-form/sign-up-form.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { EmailInput } from 'components/ui/email-input';
+import { PasswordInput } from 'components/ui/password-input';
+import { SubmitButton } from 'components/ui/submit-button';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import Users from 'constants/db';
+import AppPaths from 'constants/app-paths';
+import { useNavigate } from 'react-router-dom';
+
+interface IForm {
```

Comment:
the same with I
and also u can check this interface with the previous one, I think u can create base interface with common fields and then just use extends

Reply:
i have done it